src/NServiceBus.AmazonSQS/MessagePump.cs

```diff
@@ -305,8 +305,7 @@ async Task ProcessMessage(Message receivedMessage, CancellationToken token)
                         messageId = nativeMessageId;
                     }
 
-                    transportMessage = JsonConvert.DeserializeObject<TransportMessage>(receivedMessage.Body);
-
+                    transportMessage = SimpleJson.DeserializeObject<TransportMessage>(receivedMessage.Body);
```

Comment:
we don't need to use the strategy here since we only care about V1 backwards compat serialization for outgoing messages